### DISCUSSION TITLE
Generate paired day/night images?

### DISCUSSION THREAD

**Discussion by NothingIknow** [carla-simulator/carla#5413] [Q&A]
Thanks for reading this question:)

I want to generate **paired** day/night images. In other words, all the objects (vehicles and pedestrians) should have the same route in the day and night simulations.

I have two ideas:
1) Render daytime and record the route of each object, then render the nighttime according to that record.
2) Render the daytime and nighttime at the same time, with two simulators.

It looks like plan 2 is easier than plan1. However, I am a rookie and I am not sure if it is possible to render day and night at the same time. If plan 2 is impossible, is plan 1 possible? Is it possible to record the route of each object?

Any suggestion will be appreciated!

**Comment by beratersari** [comment]
did you solve that?
